### ISSUE TITLE
Store revert commit hashes

### ISSUE THREAD

**Issue by benthayer** [benthayer/git-gud#100]
After completing #91, we should also make an effort to do this. Similar to the other related issues, referenced in #87, we would add, as an example, `"1-": "a7f8..." to the json file.

**Comment by benthayer** [comment]
We're using a new system for tracking: It is now a CSV file, with columns `old_hash,new_hash,type`

The type for this will be "revert". We should use the `post-commit` hook to look int the commit message and get this information.

Example:
```
Revert "Initial Commit"

This reverts commit 335386491fbc848212c426a65e5b81f01ae4a3d1.                                                            
```

Detection shouldn't rely on the `Revert "X"` syntax because users might accidentally edit it, but you should rely on the second line.

**Comment by benthayer** [comment]
Once this is done, we should add revert syntax to `level_file_templates/test.spec`

**Comment by benthayer** [comment]
We are no longer doing this. We are using diff-based revert detection when we call `git gud test`. This is simpler and more clear. 

Making a new issue for adding revert syntax #209
